src/lib/BarStore.js

```diff
@@ -25,6 +25,10 @@ export default class BarStore {
     return createIntent(null, 'CLAUDY', 'io.cozy.settings', data)
   }
 
+  getSupportIntent (data) {
+    return createIntent(null, 'SUPPORT', 'io.cozy.settings', data)
```

Comment:
Just a remark: it feels weird and I am still pretty sure that it is not the way we should use intents. Poke @y-lohse

Reply:
What seems weird to you? I don't see another way to do it right now

Reply:
That's the problem, we have no other way to do this, so let's keep it. But asking for a `SUPPORT` action on a `io.cozy.settings` doctype (with no id, I guess) really means nothing to me.

Reply:
I agree with your point, maybe we could discuss about that later with the team.